discover/analytics-cloud/articles/01-what-is-liferay-analytics-cloud.markdown

```diff
@@ -0,0 +1,124 @@
+(Analytics Cloud) integrates contacts from your Liferay DE instances and CSV
+files into a single view. Analytics Cloud merges contact data into rich profiles
+of *Individuals*---unique contacts (people). Analytics Cloud lets you name
+contact fields your way, making them most useful to you and your company.
+
+Analytics Cloud lets you aggregate Individuals into *Segments*. Segments are
+aggregates of Individuals that have common characteristics. You can create
+Segments dynamically based on criteria or statically (comprising hand selected
+Individuals). You can use industry, for example, as a criteria. Industry Segment
+Analytics shows industries engaging with you most and identifies your content
+that interests them. 
+
+Contact profiles are a big part of Analytics Cloud, but analytics about people's
+interaction with your website pages and assets is a bigger part. This type of
+analytics (called Touchpoint Analytics) helps you determine content
+effectiveness in achieving your business goals. 
+
+## Touchpoint Interaction
+
+*Touchpoints* are the locations (pages) where people interact with content. Here
+are some of the Touchpoint Analytics: 
+
+-   Visitor count: the number of users (guest and signed in) visiting a 
+    Touchpoint
+-   Session length: amount of time people are spending on a channel (e.g., 
+    Liferay DE instance). 
+-   Number of sessions: number of different people at a Touchpoint
+-   Time on page: amount of time people are spending at a Touchpoint
+-   Bounce rate: percentage of visitors to a particular website who navigate 
+    away from the site after viewing only one Touchpoint
+
+*Assets* are specific types of content entities on Touchpoints. Traditional web 
+analytics are based on static websites---everyone sees the same page content. By
+additionally providing analytics on content articles, Analytics Cloud helps you
+determine the effectiveness of those articles. Asset Analytics also provides
+insight into people's interaction with dynamic personalized content. It
+generates charts on target audiences engage the content.
+
+Liferay Analytics Cloud release 1.0 supports these Asset types from Liferay
+Digital Enterprise:
+
+- Blogs
+- Documents and Media
+- Forms
+
+Regarding user behavior with a form, for example, Analytics Cloud shows the
+number of views and submissions, the percentage of users abandoning the form,
+and the form's average completion time. A line graph shows the number of views
+over time.
+
+![Figure 1: Analytics Cloud shows you how users interact with your forms.](../images/intro-form-behavior-card.png)
+
+As you view Touchpoint and Asset Analytics you can discern whether an action is
+positive or negative. Here are some examples. 
+
+Positive:
+
+- staying active in sessions
+- frequent sign in
+- submitting a form 
+- giving an article five stars
+
+Negative:
+
+- signing out shortly after signing in
+- passing by key strategic content
+- abandoning a form
+- letting a session timeout
```

Comment:
"timeout" is like "setup". When used as a verb, it's two words: time out. When used as a noun, it's one: timeout.